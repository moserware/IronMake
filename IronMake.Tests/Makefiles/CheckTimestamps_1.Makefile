# Make sure that file timestamps are checked

all: C 

C: B
	@echo making C
	@echo C contents > C

B: A
	@echo making B
	@echo B contents > B

A:
	@echo making A
	@echo A contents > A
