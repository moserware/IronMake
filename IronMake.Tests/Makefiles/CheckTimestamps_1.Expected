making A
making B
making C
